syncd/SaiSwitch.cpp

```diff
@@ -744,6 +744,25 @@ std::set<sai_object_id_t> SaiSwitch::getWarmBootDiscoveredVids() const
     return m_warmBootDiscoveredVids;
 }
 
+void SaiSwitch::redisSaveInternalOids(_In_ sai_object_id_t rid) const
+{
+    SWSS_LOG_ENTER();
+
+    std::set<sai_object_id_t> coldVids;
+
+    sai_object_id_t vid = m_translator->translateRidToVid(rid, m_switch_vid);
+
+    coldVids.insert(vid);
+
+    m_client->setDummyAsicStateObject(vid);
+
+    m_client->saveColdBootDiscoveredVids(m_switch_vid, coldVids);
```

Comment:
when doing discovery on switch when switch is created, all those "default" objects are already discovered and put as cold vids to the database, why this is needed again ?

Reply:
@kcudnik Closed as per below comments